proxy/src/main/java/com/velocitypowered/proxy/server/VelocityRegisteredServer.java

```diff
@@ -59,9 +60,7 @@
 import org.checkerframework.checker.nullness.qual.NonNull;
 import org.checkerframework.checker.nullness.qual.Nullable;
 
-/**
- * Represents a server registered on the proxy.
- */
+/** Represents a server registered on the proxy. */
```

Comment:
javadoc

Reply:
change forced by google code style

Reply:
> change forced by google code style

Please study the code style parts Google it yourself rather than using automatic formatting. Velocity does not use the full Google code style. The problem with lambda expressions is also on this list, make them the same, as in default Velocity.

Reply:
More specifically, you can check the config here https://github.com/PaperMC/Velocity/blob/dev/3.0.0/config/checkstyle/checkstyle.xml

If you're using IntelliJ, you can configure the checkstyle plugin to use that config instead